book/website/project-design/sdp/pd-sdp-sensitive-code.md

```diff
@@ -0,0 +1,42 @@
+(pd-sdp-sensitive-code)=
+
+# Sharing your Jupyter notebook
+
+One neat way of sharing your research workflow - whether that is your data cleaning process, the statistical modelling that you have applied to your data set, or how you have visualised your results  - is through [Jupyter notebooks](https://hub.gke2.mybinder.org/user/ipython-ipython-in-depth-yx75f3x5/notebooks/examples/Notebook/Notebook%20Basics.ipynb).
```

Comment:
Note: this Jupyter Notebook link does not work for me.

Reply:
Ooh thanks for flagging! Will sort it 👍